### PR TITLE
Update spanish.js

### PR DESCRIPTION
**Pull request by lawrencek1992**
*Remove all but one instance of {state}:
--I decided to leave the instance of {state} in the title section on like 138, because I thought that "COVID-19 Vaccine Spotter For Your State" as the title of the page sounded a little weird. Also the description below it (line 139) says "A tool to help you search for open appointments for the COVID-19 in pharmacies in your state. Updated every minute." I think since the abbreviation is immediately followed by text that says "in your state," it will still make sense. If you think this isn't the right judgement call, let me know, and I'll change it. Should be easy to reach for the next 4-5hr. 

*Update news section.

### CODE REVIEW

File: website/lang/spanish.js [hunk 1]
```diff
@@ -1,155 +1,153 @@
-export default {
-  metadata: {
-    title: "Observador de vacunas para COVID-19",
-    welcome: "Bienvenidos",
-    description:
-      "Una herramienta para ayudarlo de rastrear citas para la vacuna COVID-19 en las farmacias de su estado. Actualizado cada minuto.",
-    longDescription:
-      "En lugar de buscar en el sitio web de cada farmacia, escanearemos automáticamente los sitios web de las farmacias y le mostraremos las citas disponibles que podamos encontrar.",
+export default metadata: {
```
Comment (patrickdemers6) [inline website/lang/spanish.js]: One object should be exported. I think you lost the opening bracket.

File: website/lang/spanish.js [hunk 2]
```diff
@@ -1,155 +1,153 @@
...
   },
-  buttons: {
-    newAppointments: "Consultar nuevas citas",
-    visitWebsite: "Visitar el sitio de web de {name}",
+},
+time: {
+  unknown: "Hora desconocida",
+},
+appointments: {
+  available: "Citas disponibles a partir de {date_time}",
```
Comment (patrickdemers6) [inline website/lang/spanish.js]: This {date_time} shouldn't be here. I think your branch is a couple of days out of date?

**Review by patrickdemers6 — COMMENTED** [review]
I think you did these changes on old code. There should also be a new "donate" section towards the bottom that isn't there. Have you figured out how to rebase from my branch yet?

File: website/lang/spanish.js [hunk 3]
```diff
@@ -1,3 +1,34 @@
...
+Security
+Insights
+covid-vaccine-spotter/website/lang/es.js /
+@patrickdemers6
+patrickdemers6 update lang files to be named by iso 639-1 standard
+Latest commit 17f2b2a 4 hours ago
+ History
+ 1 contributor
+199 lines (199 sloc)  10.6 KB
+
```
Comment (patrickdemers6) [inline website/lang/spanish.js]: Delete.

File: website/lang/spanish.js [hunk 4]
```diff
@@ -143,13 +197,46 @@ export default {
...
+Privacy
+Security
+Status
+Docs
+Contact GitHub
+Pricing
+API
+Training
+Blog
+About
```
Comment (patrickdemers6) [inline website/lang/spanish.js]: Delete.

**Review by patrickdemers6 — CHANGES_REQUESTED** [review]

File: website/lang/spanish.js [hunk 5]
```diff
@@ -28,6 +28,26 @@ export default {
...
+        options: ["Todas las dosis", "Segunda dosis solamente"],
+      },
+      vaccineType: {
+        title: "Tipo de vacuna",
+        options: [
+          "Todos",
+          "Johnson and Johnson",
+          "Moderna",
+          "Pfizer",
+          "Unknown",
```
Comment (patrickdemers6) [inline website/lang/spanish.js]: Should "Unknown" be something different in spanish?

File: website/lang/spanish.js (hunk 4 above)
Comment (lawrencek1992) [reply website/lang/spanish.js]: Deleted that. 

I think you were right that I was working with old code, I copied your code and edited that for the {state} to say "your state"

File: website/lang/spanish.js (hunk 5 above)
Comment (lawrencek1992) [reply website/lang/spanish.js]: Yes,  that escaped my attention. Just fixed it.

**Comment by patrickdemers6** [comment]
Could you change the file name to `es.js`? I did that this afternoon on my branch. It's saying it can't merge and I'm hoping that's the issue!

File: website/lang/spanish.js (hunk 3 above)
Comment (lawrencek1992) [reply website/lang/spanish.js]: Also deleted.

**Comment by lawrencek1992** [comment]
> Could you change the file name to `es.js`? I did that this afternoon on my branch. It's saying it can't merge and I'm hoping that's the issue!

Yes. Also changed the english.js to en.js cause I see that's different on yours as well.

**Comment by patrickdemers6** [comment]
Well, I pushed and it merged lol. Thank you for all your hard work!!

**Comment by lawrencek1992** [comment]
> Well, I pushed and it merged lol. Thank you for all your hard work!!

YES! Awesome. So glad to hear that. I'm still reading about rebasing. It confuses me. If you have a favorite resource to learn about it, let me know. I very clearly need to learn that skill.

**Comment by patrickdemers6** [comment]
You're just fine! I was talking with my friend today and he said he avoids rebasing. Instead, he just merges the source's changes back in. I'm not sure what the right answer is. I should probably learn more about it as well!

I absolutely love videos by Maximilian Schwarzmüller. If you're ever looking for a crash course on various web technologies, he's the man. I'm going to watch [this](https://www.youtube.com/watch?v=CRlGDDprdOQ) video about merge vs rebase when I have time!

**Pull request merged**